simple/src/encapusl/Empolyes.java

```diff
@@ -0,0 +1,21 @@
+package encapusl;
+
+public class Empolyes {
```

Comment:
package and class name should be meaningful

Reply:
Done